### DISCUSSION TITLE
where is the activity_main.xml in xdrip?

### DISCUSSION THREAD

**Discussion by swiatmar** [NightscoutFoundation/xDrip#1940] [Q&A]
Hi I try to edit the xdrip because of "chin" mistake in the app. but can find the activity_main.xml
the fitsSystemWindows atribute has some mistake to display xdrip correctly on android watches

**Comment by tolot27** [comment]
Please explain what you want to achieve. You can find all places where `fitsSystemWindows` attribute is used using GitHub [search](https://github.com/NightscoutFoundation/xDrip/search?q=fitsSystemWindows&type=code).
And what is wrong with this attribute? Please can you provide a reference?

In xDrip there is no `activity_main.xml` but a `activity_home.xml`.

**Reply by swiatmar** [reply]
@tolot27. well I am trying to find out why xdrip doesn't appear right on round android watches. and I found out there is a problem with chin: []https://developer.android.com/training/wearables/overlays/layouts#use-xml-to-compensate-for-the-chin. this is exactly what happens on android watch devices. so I am trying to edit it in android studio...

**Reply by tolot27** [reply]
Are you sure that this is also applicable to native Android?

**Reply by tolot27** [reply]
Also, it would be interesting to emulate this in AS. Please let me know if you find some emulator images running the full Android OS. That would be really helpful for debugging.

**Reply by swiatmar** [reply]
@tolot27. exactly I am going to try to emulate it in AS and than testing it also on full android.
